styles.css

```diff
@@ -77,4 +75,357 @@
   margin-left: 20px;
   margin-top: 22px;
   width: 90%;
+  font-size: 16px;
+}
+
+.projects {
+  position: relative;
+}
+
+.projects-heading {
+  margin-top: 270px;
+  color: #1a2236;
+  font-weight: 400;
+  font-size: 50px;
+  margin-left: 20px;
+}
+
+.projects-img1 {
+  margin-top: 30px;
+  height: 200px;
+  background-repeat: no-repeat;
+  background-image: url(images/Rectangle212.png);
+  background-size: cover;
+}
+
+.projects-project2 {
+  margin-top: -2px;
+  height: 200px;
+  background-repeat: no-repeat;
+  background-image: url(images/Rectangle213.png);
+  background-size: cover;
+}
+
+.projects-project3 {
+  margin-top: -2px;
+  height: 200px;
+  background-repeat: no-repeat;
+  background-image: url(images/Rectangle214.png);
+  background-size: cover;
+}
+
+.projects-project4 {
+  margin-top: -2px;
+  height: 200px;
+  background-repeat: no-repeat;
+  background-image: url(images/Rectangle215.png);
+  background-size: cover;
+}
+
+.projects-project5 {
+  margin-top: -2px;
+  height: 200px;
+  background-repeat: no-repeat;
+  background-image: url(images/Rectangle216.png);
+  background-size: cover;
+}
+
+.projects-project6 {
+  margin-top: -2px;
+  height: 200px;
+  background-repeat: no-repeat;
+  background-image: url(images/Rectangle217.png);
+  background-size: cover;
+}
+
+.projects-img2 {
+  margin-top: -2px;
+  height: 170px;
+  background-repeat: no-repeat;
+  background-image: url(images/Rectangle22.png);
+  background-size: cover;
+}
+
+.heading-blue-background {
+  color: #fff;
+  font-family: 'poppins', Arial, Helvetica, sans-serif;
+  margin-left: 20px;
+  padding-top: 20px;
+  font-size: 20px;
+  font-weight: 300;
+}
+
+.buttons {
+  margin-left: 20px;
+  padding-top: 10px;
+}
+
+.button1,
+.button2,
+.button3 {
+  background-color: #98dede;
+  padding: 2px 5px 2px 5px;
+  border: none;
+  display: inline-block;
+  gap: 5px;
+  color: #1a2236;
+  font-size: 10.33px;
+  margin-inline: 3px;
+}
+
+.rectangle {
+  margin-left: 20px;
+  padding-top: 20px;
+}
+
+.arrow {
+  position: absolute;
+  margin-top: -35px;
+  margin-left: 165px;
+}
+
+.see {
+  position: absolute;
+  margin-top: -35px;
+  margin-left: 35px;
+  font-family: 'Poppis', Arial, Helvetica, sans-serif;
+  font-weight: 500;
+  font-size: 16px;
+}
+
+.see a {
+  text-decoration: none;
+  color: #fff;
+  cursor: pointer;
+}
+
+.about-heading,
+.contact-heading {
+  color: #1a2236;
+  font-family: 'Playfair Display', Arial, Helvetica, sans-serif;
+  font-weight: 400;
+  font-size: 50px;
+  margin-left: 20px;
+  margin-top: 40px;
+}
+
+.about-me,
+.contact-para {
+  color: #1a2236;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-size: 14px;
+  margin-left: 20px;
+  margin-right: 20px;
+  padding-top: 10px;
+  letter-spacing: -1%;
+}
+
+.about-btn {
```

Comment:
- [ ] Please ensure to include some spaces between the content and the icon for the `get my cv` button and the icon should always be on the right side of the button text as shown in the Figma design

### Your Design
![image](https://user-images.githubusercontent.com/68030297/221290676-761a76c8-e2e6-44ac-a43f-f1adfa934528.png)

### Figma Design
![image](https://user-images.githubusercontent.com/68030297/221290751-012a2d37-8660-4807-9a38-d1c389cd2655.png)